##
#  File: /patches/000-disable-plugins.patch
#  Project: rclone
#  Created Date: Wednesday, January 1st 2020, 1:49:43 pm
#  Author: ElonH[EH](elonhhuang@gmail.com)
#  License: GNU General Public License v3.0 or later(http://www.gnu.org/licenses/gpl-3.0-standalone.html)
#  Copyright (C) 2019 [ElonH]
##
# refs: https://github.com/openwrt/packages/issues/10864#issuecomment-569921457
diff --git a/rclone.go b/rclone.go
index 74fde16ab..9a9ba9150 100644
--- a/rclone.go
+++ b/rclone.go
@@ -7,7 +7,7 @@ import (
 	_ "github.com/rclone/rclone/backend/all" // import all backends
 	"github.com/rclone/rclone/cmd"
 	_ "github.com/rclone/rclone/cmd/all"    // import all commands
-	_ "github.com/rclone/rclone/lib/plugin" // import plugins
+	// _ "github.com/rclone/rclone/lib/plugin" // import plugins
 )
 
 func main() {
